inc/api.php

```diff
@@ -59,7 +59,7 @@ function gwapi_send_sms($message, $recipients, $sender='', $destaddr='MOBILE')
 
     foreach($recipients_formatted as $msisdn => $tags) {
         $rec = [
-            'msisdn' => (int)$msisdn,
+            'msisdn' => gwapi_bigint($msisdn),
```

Comment:
Hm, almost... My method still beats it in this case:
`<?php echo filter_var("(+45) 1234567890 1234567890", FILTER_SANITIZE_NUMBER_INT);`
which outputs:
`+4512345678901234567890`

I don't believe the gateway supports the + character prefixed to the MSISDN?